metricbeat/module/ceph/mgr_osd_perf/_meta/data.json

```diff
@@ -0,0 +1,26 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "ceph": {
+        "mgr_osd_perf": {
```

Comment:
I guess this is something we weren't collecting from the old API. Would it still make sense to use `osd_perf` here?

Reply:
(discussed offline)

Let's keep the `mgr_` prefix to distinguish new metricsets from refreshed.